src/herbpy/tsr/fuze.py

```diff
@@ -0,0 +1,55 @@
+import numpy
+from prpy.tsr.tsrlibrary import TSRFactory
+from prpy.tsr.tsr import *
+
+@TSRFactory('herb', 'fuze_bottle', 'point')
+def point_at_obj(robot, bottle, manip=None):
+    '''
+    @param robot The robot performing the point
+    @param bottle The bottle to point
+    @param manip The manipulator to perform the grasp. if None 
+        the active manipulator on the robot is used
+    '''
+
+    if manip is None:
+        manip_idx = robot.GetActiveManipulatorIndex()
+    else:
+        with manip.GetRobot():
+            manip.SetActive()
+            manip_idx = manip.GetRobot().GetActiveManipulatorIndex()
+		
+    # compute T_ow
+    T0_w_0 = bottle.GetTransform()
+    T0_w_1 = numpy.identity(4)
+
+    # compute T_we. Tuned once and used forever onward 
+    # Insures pointer points at object with right arm
+    arm_start =  numpy.array([[ 0.07277, -0.71135,  0.69905,  0.58575],
```

Comment:
I would prefer to not add a bogus link to the URDF. It will just add more clutter to our already quite cluttered TF tree.

We have have the `/right/finger0_2`, `/right/jfinger1_2`, and `/right/finger2_2` links. The origin of these links is aligned with one axis along the axis of revolution of the distal joint and another axis pointed towards the fingertip. We should only have to apply a small offset along one axis to get the position of the fingertip.

I agree that this assumes that the hand is already in a sane preshape. We could encapsulate changing the hand preshape and planning to this TSR using @jeking04's `ActionLibrary`. See #15 for an example.